### ISSUE TITLE
errors while loading compiling tensorflow model

### ISSUE THREAD

**Issue by matkaa94** [tensorflow/tensorflow#42776]
**System information**
- Windows 10 64 bit:
- tensorflow  (v2.3) installed via pip installer
- Python v3.8.3 using Spyder4:
-CUDA: 10.1.:
- GPU: Nvidia GeForce GTX 1060 3 GB

Hello,

i am trying to compile a code from YT. In the video it worked. I noticed there are some warnings, which are probably regarded to the cudnn64_7.dll file and also some other missing parts.

Link to the Video and Code: https://pythonprogramming.net/crypto-rnn-model-deep-learning-python-tensorflow-keras/?completed=/balancing-rnn-data-deep-learning-python-tensorflow-keras/

I donwloaded Cuddn and extracted the files, as well as made a PATH systemvariable to this file.
This is the error I get:

**2020-08-29 18:45:48.778870: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.432459: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library nvcuda.dll
2020-08-29 18:45:52.566076: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1060 3GB computeCapability: 6.1
coreClock: 1.7085GHz coreCount: 9 deviceMemorySize: 3.00GiB deviceMemoryBandwidth: 178.99GiB/s
2020-08-29 18:45:52.568033: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.598770: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2020-08-29 18:45:52.635186: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cufft64_10.dll
2020-08-29 18:45:52.651335: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library curand64_10.dll
2020-08-29 18:45:52.693526: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusolver64_10.dll
2020-08-29 18:45:52.707974: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusparse64_10.dll
2020-08-29 18:45:52.708778: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cudnn64_7.dll'; dlerror: cudnn64_7.dll not found
2020-08-29 18:45:52.709174: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-08-29 18:45:52.711705: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-08-29 18:45:52.735435: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15544f7c200 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-08-29 18:45:52.735933: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-08-29 18:45:52.737008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-08-29 18:45:52.737368: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
2020-08-29 18:45:53.679101: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:45:53.680145: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1391] Profiler found 1 GPUs
2020-08-29 18:45:53.682001: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti64_101.dll'; dlerror: cupti64_101.dll not found
2020-08-29 18:45:53.682796: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti.dll'; dlerror: cupti.dll not found
2020-08-29 18:45:53.683220: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:01.571888: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:57:01.572185: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
Epoch 1/10
   1/1218 [..............................] - ETA: 0s - loss: 0.7607 - accuracy: 0.6094WARNING:tensorflow:From C:\Users\matej\anaconda3\lib\site-packages\tensorflow\python\ops\summary_ops_v2.py:1277: stop (from tensorflow.python.eager.profiler) is deprecated and will be removed after 2020-07-01.
Instructions for updating:
use `tf.profiler.experimental.stop` instead.

2020-08-29 18:45:48.778870: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.432459: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library nvcuda.dll
2020-08-29 18:45:52.566076: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1060 3GB computeCapability: 6.1
coreClock: 1.7085GHz coreCount: 9 deviceMemorySize: 3.00GiB deviceMemoryBandwidth: 178.99GiB/s
2020-08-29 18:45:52.568033: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.598770: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2020-08-29 18:45:52.635186: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cufft64_10.dll
2020-08-29 18:45:52.651335: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library curand64_10.dll
2020-08-29 18:45:52.693526: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusolver64_10.dll
2020-08-29 18:45:52.707974: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusparse64_10.dll
2020-08-29 18:45:52.708778: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cudnn64_7.dll'; dlerror: cudnn64_7.dll not found
2020-08-29 18:45:52.709174: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-08-29 18:45:52.711705: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-08-29 18:45:52.735435: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15544f7c200 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-08-29 18:45:52.735933: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-08-29 18:45:52.737008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-08-29 18:45:52.737368: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
2020-08-29 18:45:53.679101: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:45:53.680145: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1391] Profiler found 1 GPUs
2020-08-29 18:45:53.682001: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti64_101.dll'; dlerror: cupti64_101.dll not found
2020-08-29 18:45:53.682796: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti.dll'; dlerror: cupti.dll not found
2020-08-29 18:45:53.683220: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:01.571888: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:57:01.572185: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:07.812968: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:57:07.813318: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
   2/1218 [..............................] - ETA: 4:13 - loss: 0.8007 - accuracy: 0.5312WARNING:tensorflow:Callbacks method `on_train_batch_end` is slow compared to the batch time (batch time: 0.0698s vs `on_train_batch_end` time: 0.3459s). Check your callbacks.
  15/1218 [..............................] - ETA: 1:22 - loss: 0.8180 - accuracy: 0.4938
2020-08-29 18:45:48.778870: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.432459: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library nvcuda.dll
2020-08-29 18:45:52.566076: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1060 3GB computeCapability: 6.1
coreClock: 1.7085GHz coreCount: 9 deviceMemorySize: 3.00GiB deviceMemoryBandwidth: 178.99GiB/s
2020-08-29 18:45:52.568033: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudart64_101.dll
2020-08-29 18:45:52.598770: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cublas64_10.dll
2020-08-29 18:45:52.635186: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cufft64_10.dll
2020-08-29 18:45:52.651335: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library curand64_10.dll
2020-08-29 18:45:52.693526: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusolver64_10.dll
2020-08-29 18:45:52.707974: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cusparse64_10.dll
2020-08-29 18:45:52.708778: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cudnn64_7.dll'; dlerror: cudnn64_7.dll not found
2020-08-29 18:45:52.709174: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1753] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-08-29 18:45:52.711705: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-08-29 18:45:52.735435: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15544f7c200 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-08-29 18:45:52.735933: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-08-29 18:45:52.737008: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-08-29 18:45:52.737368: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      
2020-08-29 18:45:53.679101: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:45:53.680145: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1391] Profiler found 1 GPUs
2020-08-29 18:45:53.682001: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti64_101.dll'; dlerror: cupti64_101.dll not found
2020-08-29 18:45:53.682796: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'cupti.dll'; dlerror: cupti.dll not found
2020-08-29 18:45:53.683220: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:01.571888: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:57:01.572185: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:07.812968: I tensorflow/core/profiler/lib/profiler_session.cc:164] Profiler session started.
2020-08-29 18:57:07.813318: E tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1441] function cupti_interface_->Subscribe( &subscriber_, (CUpti_CallbackFunc)ApiCallback, this)failed with error CUPTI could not be loaded or symbol could not be found.
2020-08-29 18:57:07.943306: I tensorflow/core/profiler/internal/gpu/device_tracer.cc:223]  GpuTracer has collected 0 callback api events and 0 activity events. 
2020-08-29 18:57:08.092651: I tensorflow/core/profiler/rpc/client/save_profile.cc:176] Creating directory: logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07
2020-08-29 18:57:08.155731: I tensorflow/core/profiler/rpc/client/save_profile.cc:182] Dumped gzipped tool data for trace.json.gz to logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07\DESKTOP-KSAFH0A.trace.json.gz
2020-08-29 18:57:08.201015: I tensorflow/core/profiler/rpc/client/save_profile.cc:176] Creating directory: logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07
2020-08-29 18:57:08.206548: I tensorflow/core/profiler/rpc/client/save_profile.cc:182] Dumped gzipped tool data for memory_profile.json.gz to logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07\DESKTOP-KSAFH0A.memory_profile.json.gz
2020-08-29 18:57:08.220482: I tensorflow/python/profiler/internal/profiler_wrapper.cc:111] Creating directory: logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07Dumped tool data for xplane.pb to logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07\DESKTOP-KSAFH0A.xplane.pb
Dumped tool data for overview_page.pb to logs/.format(NAME)\train\plugins\profile\2020_08_29_16_57_07\DESKTOP-KSAFH0A.overview_page.pb**

Can somebody help me please??

Thanks in advance and BR!

**Comment by ravikyram** [comment]
@matkaa94 

Just to verify did you follow the instructions as mentioned in the [website](https://www.tensorflow.org/install/source_windows).
Please, see tested build configurations from [here](https://www.tensorflow.org/install/source_windows#gpu).Thanks!

**Comment by matkaa94** [comment]
@ravikyram 

Thanks i had some issues with cupti and cuda, but somehow managed it.
